fastapi/README.md

```diff
@@ -12,5 +12,8 @@ based on selections.
 
 
 ## Usage
-* `cruft create . --directory="fastapi"` to use from source
+* `poetry init` from `templates` folder.
+* `poetry add cruft`
+* `cruft create . --directory="fastapi"` to use from source (current commit)
+  * `cookiecutter fastapi` to use from current state (used while testing uncommitted changes)
 * `cruft create https://github.com/Intility/templates.git --directory="fastapi"` to use from GitHub
```

Comment:
are all these steps necessary to init directly from github? cloning the repo first and all?

Reply:
These steps are for when editing source (like I'm doing right now). 

I'll make it more clear.😊